### PR TITLE
Make file tree the same as in our others repos with dockerfile

### PR DESCRIPTION
**Pull request by dynbit**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          exclusive: true
+      - run:
+          name: Build image
+          command: |
+            docker build -t circleci-docker .
+      - deploy:
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              docker login -u $DOCKER_USER -p $DOCKER_PASS
+              docker tag circleci-docker:latest vlcty/circleci-docker:$CIRCLE_BUILD_NUM
```
Comment (pranas) [inline circle.yml]: for consistent naming, use ``-docker`` only on repos, omit from the image names, like the others ``caffe-ubuntu`` image, ``caffe-ubuntu-docker`` repo name

Comment (dynbit) [reply circle.yml]: Updated. Take a look now!

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - image: docker:git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          reusable: true
+          exclusive: true
+      - run:
+          name: Build image
+          command: |
+            docker build -t circleci-docker .
```
Comment (pranas) [inline circle.yml]: leftover, will fail

**Review by pranas — APPROVED** [review]

File: circle.yml (hunk 2 above)
Comment (dynbit) [reply circle.yml]: 👍  good catch!

**Pull request merged**